index.d.ts

```diff
@@ -1,30 +1,30 @@
 import * as Electron from 'electron';
-
+import * as Main from 'electron/main'
 // Taken from `RemoteMainInterface`
 export var app: Electron.App;
 export var autoUpdater: Electron.AutoUpdater;
 export var BrowserView: typeof Electron.BrowserView;
 export var BrowserWindow: typeof Electron.BrowserWindow;
```

Comment:
Confused why not update all of these?

```suggestion
export var BrowserView: Main.BrowserView;
export var BrowserWindow: Main.BrowserWindow;
```

etc.

Reply:
Thank you for your review.

I tried.
> export var BrowserWindow: Main.BrowserWindow;

However, an error occurs because the static member of BrowserWindow cannot be referenced.


code&error
```
import * as _remote from '@electron/remote';
_remote.BrowserWindow.getAllWindows();
                      ~~~~~~~~~~~~~~~
                      ^ Property 'getAllWindows' is a static member of type 'BrowserWindow'.

```